docs/source/datastructures_peak.rst

```diff
@@ -53,6 +49,28 @@ First we create a spectrum and insert peaks with descending mass-to-charge ratio
     1200.0 1.0
     1300.0 1.0
     1400.0 1.0
+
+    First peak: 500.0 1.0
+
+
+Note how lines 11-12 (as well as line 19) use the direct access to the
+``Peak1D`` objects (explicit iteration through the ``MSSpectrum`` object, which
+is convenient but slow since a new ``Peak1D`` object needs to be created each
+time).
+The following example uses the faster access through numpy arrays with .. py:function:: get_peaks() or
+.. py:function:: set_peaks(). Direct iteration is only shown for demonstration purposes and should not be used in
```

Comment:
```suggestion
The following example uses the faster access through numpy arrays with :py:meth:`~.MSSpectrum.get_peaks` or
:py:meth:`~.MSSpectrum.set_peaks`. Direct iteration is only shown for demonstration purposes and should not be used in
```

Per the documentation that I linked:
- .. py.*** is for definitions
- :py:*** is for references
- func is for functions. Those are methods -> meth
- use "~" prefix to not spell the fully qualified name